src/components/Modals/VideoModal/index.jsx

```diff
@@ -19,7 +19,7 @@ function Modal({ isOpen, onClose, videoLink = VIDEO_LINK }) {
     window.addEventListener('keydown', close)
 
     return () => window.removeEventListener('keydown', close)
-  }, [])
+  })
```

Comment:
Are you sure that this fix is ​​really needed?

Reply:
I've checked - no. I've made it, because I've got EsLint warning, while comitting and took it for an error

Reply:
Should I reset?

Reply:
it is not very good to try to do something else in your task that is not directly related to your task